docs/shapes.rst

```diff
@@ -0,0 +1,352 @@
+.. include:: references.txt
+
+.. _sh:
+
+.. _sh-shapes:
+
+Shapes
+======
+
+This section shows one example how to construct a region for each shape that's
+currently supported.
+
+* `~regions.CircleSkyRegion` and `~regions.CirclePixelRegion`
+
+.. code-block:: python
+
+    >>> from astropy.coordinates import Angle, SkyCoord
+    >>> from regions import PixCoord, CircleSkyRegion, CirclePixelRegion
+
+    >>> circle_sky = CircleSkyRegion(center=SkyCoord(42, 43, unit='deg'),
```

Comment:
these `SkyCoord` objects all need a `frame` to be specified, e.g., `frame='fk5'`.  The code might be functional without that, but it's nonsense.

Reply:
default is `ICRS` but it's better to make it explicit.